definitions/tools/pizzly.cwl

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "pizzly gene fusion detector"
+baseCommand: ["pizzly"]
+requirements:
+    - class: InlineJavascriptRequirement
+    - class: ResourceRequirement
+      coresMin: 1
+      ramMin: 16000
+    - class: DockerRequirement
+      dockerPull: "chrisamiller/docker-pizzly"
```

Comment:
not urgent, but we should probably move this image out of my personal space and into mgibio.